backend/pennmobile/settings/production.py

```diff
@@ -20,7 +20,7 @@
 
 # Sentry settings
 SENTRY_URL = os.environ.get("SENTRY_URL", "")
-sentry_sdk.init(dsn=SENTRY_URL, integrations=[DjangoIntegration()])
+sentry_sdk.init(dsn=SENTRY_URL, integrations=[CeleryIntegration(), DjangoIntegration()])
 
 # DLA settings
 PLATFORM_ACCOUNTS = {"ADMIN_PERMISSION": "penn_mobile_admin"}
```

Comment:
Since you're using Redis in prod now, you might as well use it for [Django's cache backend](https://docs.djangoproject.com/en/4.0/topics/cache/):

```
CACHES = {
    "default": {
        "BACKEND": "django_redis.cache.RedisCache",
        "LOCATION": REDIS_URL,
        "OPTIONS": {
            "CLIENT_CLASS": "django_redis.client.DefaultClient",
        },
    }
}
```

([source](https://github.com/pennlabs/penn-courses/blob/master/backend/PennCourses/settings/production.py#L32) in PCx settings).

Then if you actually use caching in Django, you should add the following to settings.development as well:

```
CACHES = {
    "default": {
        "BACKEND": "django.core.cache.backends.dummy.DummyCache",
    }
}
```

([source](https://github.com/pennlabs/penn-courses/blob/master/backend/PennCourses/settings/development.py#L16) in PCx settings). Note that we use dummy caching in dev even though we have Redis in our dev environment (for Celery), to enforce that we don't actually rely on Django's caching for important storage (i.e. the Django cache can be purged at any time, causing possible slowdowns, but without causing any change in semantics).

For an example of how we purge Django cache entries while leaving the Celery part of the cache alone, see [here](https://github.com/pennlabs/penn-courses/blob/master/backend/review/management/commands/clearcache.py).